libraries/rush-lib/src/api/CommandLineConfiguration.ts

```diff
@@ -673,12 +672,22 @@ export class CommandLineConfiguration {
     (this.additionalPathFolders as string[]).unshift(pathFolder);
   }
 
+  /**
+   * This function replaces colons (":") with underscores ("_").
+   *
+   * ts-command-line restricts command names to lowercase letters, numbers, underscores, and colons.
+   * Replacing colons with underscores produces a filesystem-safe name.
+   */
+  private _normalizeNameForLogFilenameIdentifiers(name: string): string {
+    return name.replace(/:/g, '_'); // Replace colons with underscores to be filesystem-safe
+  }
```

Comment:
Since we bundle rush-lib using webpack and this function does not reference `this`, it being a free function is more efficient.